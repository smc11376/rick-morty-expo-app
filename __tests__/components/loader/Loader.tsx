import { render, screen } from "@testing-library/react-native";
import Loader from "@/components/loader";

test("Loader renders properly", () => {
  render(<Loader />);

  expect(screen.toJSON()).toMatchSnapshot();
});
