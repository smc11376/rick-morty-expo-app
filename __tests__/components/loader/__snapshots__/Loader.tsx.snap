// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders properly 1`] = `
<ActivityIndicator
  color="white"
  size="large"
  style={
    {
      "justifyContent": "center",
      "marginVertical": 50,
    }
  }
/>
`;
